crates/host/src/wasmbus/mod.rs

```diff
@@ -2504,3 +2596,75 @@ impl Host {
         }
     }
 }
+
+// TODO: use a better format https://github.com/wasmCloud/wasmCloud/issues/508
+#[derive(Serialize, Deserialize)]
+struct StoredClaims {
+    call_alias: String,
+    #[serde(rename = "caps")]
+    capabilities: String,
+    contract_id: String,
+    #[serde(rename = "iss")]
+    issuer: String,
+    name: String,
+    #[serde(rename = "rev")]
+    revision: String,
+    #[serde(rename = "sub")]
+    subject: String,
+    tags: String,
+    version: String,
+}
+
+impl TryFrom<jwt::Claims<jwt::Actor>> for StoredClaims {
+    type Error = anyhow::Error;
+
+    fn try_from(claims: jwt::Claims<jwt::Actor>) -> Result<Self, Self::Error> {
+        let issuer = claims.issuer.clone();
+        let subject = claims.subject.clone();
```

Comment:
Aren't these clones redundant?

You could even destruct this as `let jwt::Claims{ issuer, subject, metadata } = claims`, right? (or even do that in the function parameter)

Reply:
Yep, should have cleaned this up after iterating 👍